<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingShrinkHighLowDialog</class>
 <widget class="QDialog" name="SettingShrinkHighLowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>111</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxSkipDot">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>111</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Skip dot</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>451</width>
     <height>221</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>LayerName</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Method</string>
    </property>
   </column>
  </widget>
  <widget class="QCheckBox" name="checkBoxHorizontalTile">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>0</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Horizontai tile</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxVerticalTile">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Vertical tile</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
